spec/requests/sales_logs_controller_spec.rb

```diff
@@ -186,31 +186,31 @@
           end
 
           context "with year filter" do
-            let!(:sales_log_2021) do
+            let!(:sales_log_2022) do
```

Comment:
now we only allow logs from the 22/23 collection year, filtering by years doesn't really make as much sense but I've left it in as you can see